components/package.json

```diff
@@ -1,27 +1,27 @@
 {
-    "name": "@pxblue/react-components",
+    "name": "@brightlayer-ui/react-components",
     "version": "5.3.1",
```

Comment:
"version": "5.3.1-beta.0"

Once we have a beta package published, could you make a follow-up PR to update the storybook app using beta package?